package.json

```diff
@@ -42,6 +42,7 @@
 		"@nextcloud/vue": "^5.1.0",
 		"camelcase": "^6.3.0",
 		"debounce": "^1.2.1",
+		"lodash": "^4.17.21",
```

Comment:
Please refer to this documentation to avoid using lodash: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore